lib/tasks/create_fake_data.rake

```diff
@@ -0,0 +1,93 @@
+namespace :db do
+  namespace :seed do
+    desc 'Generate fake patient entries for data wranglers'
+    task :create_fake_data => :environment do
+
+        Patient.destroy_all
+
+        users = User.all
+        clinics = Clinic.all
+
+        gen = Random.new(2020) # our random number generator: https://ruby-doc.org/core-2.7.0/Random.html
+
+        5000.times do |idx|
+          flag = gen.rand < 0.05 # gen.rand will always return a float between 0 and 1 
```

Comment:
Let's put an additional logging statement in, since making 5k patients takes a little while:
```suggestion
          idx % 10 == 0 ? print("\nMaking fake pt #{idx} of 5000") : print('.')
          flag = gen.rand < 0.05 # gen.rand will always return a float between 0 and 1 
```

Reply:
maybe replace '5000' with a variable for to DRY?